/*
    PARA COMPILAR USE O COMANDO (NA PASTA src)
    python -m grpc_tools.protoc -I./protos --python_out=. --grpc_python_out=. ./protos/loginuser.proto    
*/

syntax = "proto3";

// Definição do serviço
service LoginUser {
    // Recebe a página do passarinho
    rpc login (UserLogin) returns (UserInfo) {}
}

// Mensagem que retorna as informações de um usuário
message UserInfo {
    string crBio = 1;
    string name = 2;
    string password_hash = 3;
}

message UserLogin {
    string crBio = 1;
    string password = 2;
}
